common/xbps-src/shutils/build_dependencies.sh

```diff
@@ -296,9 +304,9 @@ install_pkg_deps() {
     #
     # Target run time dependencies
     #
-    if [[ ${depends} ]]; then
+    local _cleandeps=$(setup_pkg_depends "" 1 1) || exit 1
```

Comment:
because `$1` is set in `setup_pkg_depends` the 3rd argument will be skipped.
That means that here 2nd argument should be unset to make this work as expected.

Reply:
Change this to `$(setup_pkg_depends '' '' 1)`

Reply:
oh doh! I confused the args. It's ok in fact!